composer.json

```diff
@@ -17,10 +17,12 @@
         "zendframework/zend-stdlib": "~2.5"
     },
     "require-dev": {
-        "zendframework/zend-filter": "~2.5",
+        "zendframework/zend-filter": "~2.6",
         "zendframework/zend-i18n": "~2.5",
         "zendframework/zend-servicemanager": "~2.5",
         "zendframework/zend-validator": "~2.5",
+        "zendframework/zend-progressbar": "~2.5",
```

Comment:
Interestingly, in `ZendTest\Transfer\Adatper\HttpTest`. Missing requirement, evidently.